code/game/turfs/simulated/walls.dm

```diff
@@ -140,6 +140,10 @@
 	if (iscultist(user) && !(locate(/obj/effect/cult_shortcut) in src))
 		var/datum/cult_tattoo/CT = user.checkTattoo(TATTOO_SHORTCUT)
 		if (CT)
+			var/mob/living/carbon/C = user
+			if (C.muted())
```

Comment:
A more robust naming choice here might reduce confusion over people thinking this cares about your ability to speak. I know that it doesn't, but will future coders?